### PR TITLE
Improvements : Added Arrow FAQ and Logo Footer 

### LINKED ISSUE #83: fix: FAQ section
image

1. fix the width of the FAQ section 
2. add arrow down icon on every FAQ.

### PR DESCRIPTION
**Pull request by RamGoel**
Fixes #83

### CODE REVIEW

File: dev/styles/faqPage.css [hunk 1]
```diff
@@ -3,9 +3,10 @@
 .faqPage {
   min-height: 90vh;
 }
+
 .accordion {
   display: flex;
-  font-family: Poppins !important;
```
Comment (pulkit-30) [inline dev/styles/faqPage.css:8]: Is this right way....??? 
Not working for me, can you please verify this

File: dev/styles/faqPage.css [hunk 2]
```diff
@@ -21,48 +22,65 @@
...
   display: none;
 }
+
 .accordion .accordion-text li {
   list-style: none;
   cursor: pointer;
 }
+
+.accordion-button::after {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%23fff'%3e%3cpath fill-rule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z'/%3e%3c/svg%3e") !important;
```
Comment (pulkit-30) [inline dev/styles/faqPage.css:64]: Instead can we use font-awsm 

```
<i class="fa-sharp fa-solid fa-caret-down"></i>
```

File: dev/styles/footer.css [hunk 3]
```diff
@@ -7,7 +7,7 @@ footer {
 
 footer .social {
   margin: 0;
-  width: 80% !important;
+  /* width: 80% !important; */
```
Comment (pulkit-30) [inline dev/styles/footer.css:10]: Please, remove extra comment lines

File: dev/index.html [hunk 4]
```diff
@@ -36,7 +36,13 @@
...
     <link rel="stylesheet" href="./styles/contactForm.css" />
     <link rel="stylesheet" href="./styles/animation.css" />
-
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+      integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
Comment (pulkit-30) [inline dev/index.html:45]: For what?

File: dev/index.html [hunk 5]
```diff
@@ -621,22 +628,21 @@ <h3><a href="#">Lorem Ipsum </a></h3>
           <div class="accordion-text" id="accordion-box">
             <div class="title">FAQ</div>
             <ul class="faq-text" id="accordioFlushExample">
-              <li>
+              <li class="">
```
Comment (pulkit-30) [inline dev/index.html:631]: Please remove `class="" `

File: dev/index.html [hunk 6]
```diff
@@ -933,8 +938,17 @@ <h3><a href="#">Lorem Ipsum </a></h3>
...
     <!-- ---------------------------------------------------------------------------- -->
 
-    <footer class="flex">
-      <div class="social flex">
+    <div class="col-8 mx-auto p-3 bg-white d-flex w-100 justify-content-around">
+      <img src="./assets/elixirLogo.png" class="footerLogo " />
+
+      <img src="./assets/gdscLogo.png" class="footerLogo " />
+      <img src="./assets/ccLogo.png" class="footerLogo " />
+      <img src="./assets/gfgLogo.png" class="footerLogo " />
```
Comment (pulkit-30) [inline dev/index.html:946]: Add alt attribute

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by pulkit-30** [comment]
Screenshot 2022-09-04 at 8 36 53 PM

Please increase some height of the this section

File: dev/styles/faqPage.css (hunk 2 above)
Comment (RamGoel) [reply dev/styles/faqPage.css:64]: As we have used bootstrap accordions, it adds a down arrow by default, but the colour of the arrow doesn't match the background, so this has been done to change the colour.

**Comment by RamGoel** [comment]
@pulkit-30 @dewank07 Check if everything Okay.

File: dev/styles/faqPage.css (hunk 1 above)
Comment (pulkit-30) [reply dev/styles/faqPage.css:8]: @RamGoel mind having a look at this

Comment (RamGoel) [reply dev/styles/faqPage.css:8]: It is supposed to work now, working on my side. Confirm, please. @dewank07 @Anv3sh

**Review by pulkit-30 — APPROVED** [review]
GTG 🚀

**Pull request merged**